docs/usage/configuration-options.md

```diff
@@ -2446,6 +2446,18 @@ In those cases a feature request needs to be implemented.
 !!! warning "Warning for Maven users"
     For `minimumReleaseAge` to work, the Maven source must return reliable `last-modified` headers.
 
+    If your custom maven source registry doesnt support `last-modified` header, like GAR (Google Artifact Registry's Maven implementation, then feasible option is to extend maven source registry url with https://repo1.maven.org/maven2 as the first item, then `currentVersionTimestamp` via `last-modified` will be taken from maven central for public dependencies.
+    
+    <!-- markdownlint-disable MD046 -->
```

Comment:
I disabled the rule, because it seems to be false negative. because it expects fenced code, and it is, but it thinks its indented
```
MD046/code-block-style Code block style [Expected: fenced; Actual: indented] https://github.com/DavidAnson/markdownlint/blob/v0.37.4/doc/md046.md
```